src/robusta/core/reporting/blocks.py

```diff
@@ -434,3 +436,41 @@ def grade(self):
             return "E"
         else:
             return "F"
+
+
+class EventsBlock(TableBlock):
+    """
+    Table display of a Events, Persists the events on the Robusta Platform.
+
+    Note: Wider tables appears as a file attachment on Slack, because they aren't rendered properly inline
+    """
+
+    namespace: str
```

Comment:
Why do we need it here?
We always have it on the finding
It's not a part of the enrichment really, is it?